src/bp-core/bp-core-update.php

```diff
@@ -1191,6 +1195,70 @@ function bb_update_to_1_5_6() {
 }
 
 /**
+ * x.x.x update routine.
```

Comment:
@asaquzzamanbb update x.x.x to proper version

Reply:
Done here https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/2de904dabcd9f83f687eb4fcff2b522667b00d93